enrolments/serializers.py

```diff
@@ -10,3 +10,22 @@ class Meta:
             "season",
             "year",
         ]
+
+
+class ClassListSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Class
+        fields = ["id", "name", "facilitator_id"]
+
+
+class SessionDetailsSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
nit: can we name this `SessionDetailSerializer`?

Reply:
now that i look again, i don't think it's needed! the tests still work without the serializer, so i think i will remove it